frontend/app/assets/javascripts/spree/frontend/checkout/address.js

```diff
@@ -127,3 +127,17 @@ Spree.ready(function ($) {
   }
   Spree.onAddress()
 })
+
+$(document).ready(function() {
```

Comment:
It might be better to move this code to Spree.ready{} above. (because of the consistency and turbolinks)

![Screenshot 2020-02-12 at 17 01 34](https://user-images.githubusercontent.com/10231930/74352813-5d6e0000-4db9-11ea-9993-e02069ccf82a.png)